packages/docusaurus-plugin-openapi/src/sidebars/index.ts

```diff
@@ -5,38 +5,97 @@
  * LICENSE file in the root directory of this source tree.
  * ========================================================================== */
 
-import { ApiMetadata, ApiPageMetadata } from "../types";
+import path from "path";
+
+import _ from "lodash";
+
+import type { PropSidebarItem } from "../types";
```

Comment:
nit: `PropSidebar` is the array form of `PropSidebarItem` so maybe we should use that instead since we only reference this as an array?